include/sentry.h

```diff
@@ -1499,6 +1499,15 @@ SENTRY_API void sentry_options_set_system_crash_reporter_enabled(
 SENTRY_API void sentry_options_set_crashpad_wait_for_upload(
     sentry_options_t *opts, int wait_for_upload);
 
+/**
+ * Enables or disables console logging during crash handling.
+ * When disabled, Sentry will not invoke logger callbacks during crash
+ * handler execution. This can be useful to avoid potential issues during
+ * crash handling. This is enabled by default.
+ */
+SENTRY_API void sentry_options_set_handler_logging_enabled(
+    sentry_options_t *opts, int enabled);
+
```

Comment:
Seems to belong to the logger configuration options section. 
Maybe consider renaming to `void sentry_options_set_logger_is_signal_safe(sentry_options_t *opts, int value)`.